Flow.Launcher/App.xaml.cs

```diff
@@ -61,6 +61,8 @@ private async void OnStartupAsync(object sender, StartupEventArgs e)
                 _settingsVM = new SettingWindowViewModel(_updater, _portable);
                 _settings = _settingsVM.Settings;
 
+                Http.Proxy = _settings.Proxy;
```

Comment:
Set http proxy first so that when the plugin manager initializes it will follow the proxy.